Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM amazonlinux:2023 AS base
 ENV NODE_VERSION=24.4.0
 
 RUN yum update -y && \
-    yum install -y tar xz openssl && \
+    # Amazon Linux 2023 base image is minimal and does not include curl by default,
+    # so we explicitly install curl here to ensure availability.
+    yum install -y tar xz openssl curl && \
```

Comment:
I did a quick test to see if curl was available in the base image. It seems like it is.

```dockerfile
FROM amazonlinux:2023
CMD ["curl", "--version"]
```

Tested it with:

```sh
docker build -t curl-test .
docker run curl-test
```

Which produces the output:

```sh
curl 8.11.1 (aarch64-amazon-linux-gnu) libcurl/8.11.1 OpenSSL/3.2.2 zlib/1.2.11 libidn2/2.3.2 libpsl/0.21.5 nghttp2/1.59.0
Release-Date: 2024-12-11
Protocols: file ftp ftps http https ipfs ipns
Features: alt-svc AsynchDNS GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz PSL SPNEGO SSL threadsafe UnixSockets
```